src/Utils/dialogUtils.test.ts

```diff
@@ -259,12 +257,12 @@ describe('dialogUtils', () => {
         test('found', () => {
             const trainDialog1C = copyTrainDialog()
             let result = findMatchingTrainDialog(trainDialog1C, [trainDialog1, trainDialog2, trainDialog3])
-            expect(result?.trainDialogId).toEqual(trainDialog1.trainDialogId)
+            expect(result && result.trainDialogId).toEqual(trainDialog1.trainDialogId)
```

Comment:
Was the previous new syntax not working here ?

Reply:
Merge issue.  Will fix